src/hp/netcheck.rs

```diff
@@ -1567,6 +1817,7 @@ mod tests {
             .get_report(dm, None, None)
             .await
             .context("failed to get netcheck report")?;
+        dbg!(&r);
```

Comment:
stray `dbg`

Reply:
oh, this was on purpose... do you mind leaving these in or would rather have them removed?  It's much nicer when running the test with --nocapture.